src/storage/mod.rs

```diff
@@ -1700,6 +1703,7 @@ mod tests {
                     Mutation::Put((Key::from_raw(b"b"), b"bb".to_vec())),
                     Mutation::Put((Key::from_raw(b"c"), b"cc".to_vec())),
                 ],
+                vec![false; 3],
```

Comment:
@nrc Sorry, there're still a number of places where providing all-false is_pessimistic_locks is not necessary. Would you please remove all of them?